ts/Series/Treemap/TreemapNode.ts

```diff
@@ -0,0 +1,55 @@
+import type TreemapSeries from './TreemapSeries.js';
+import type BBoxObject from '../../Core/Renderer/BBoxObject.js';
+import type TreemapPoint from './TreemapPoint.js';
+
+namespace TreemapNode {
```

Comment:
I am now wondering - may we borrow something from the NodesComposition?

Reply:
The node structure is different. Could you pinpoint some common functionalities, that could be easily transferred?